docs/templates/partner-plugin-template/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+nav_title: # Set a title for the navigation menu
+---
+
+## How it works
+
+<!--How does your plugin work? -->
+
+## How to install
+
+<!-- If your plugin can be installed via luarocks, change {YOUR_PLUGIN_NAME} to your own plugin, e.g. example-plugin.
+If the plugin is an integration in your catalog, or is installed in some other way, replace the following instructions with your own. -->
+
+The `.rock` file is a self-contained package that can be installed locally or from a remote server.
+
+If the LuaRocks utility is installed in your system (this is likely the case if you used one of the official installation packages), you can install the `rock` in your LuaRocks tree, that is, the directory in which LuaRocks installs Lua modules.
```

Comment:
should this also be removed even if they are using LuaRocks? It just reads like it's part of the instructions on how to write the plugin docs instead of how to install the plugin.